src/index.html

```diff
@@ -10,14 +10,20 @@
     <app-component></app-component>
 
     <!--
-      We load zone.js outside the bundle. That's because it's a "prollyfill"
+      We load zone.js outside the bundle. That's because it's a "pollyfill"
```

Comment:
actually that spelling is intentional, it's a portmonteau of probably and polyfill

Reply:
I think Angular repo had to add a _comment for a comment_ at some point because people kept changing it back and forth in reviews :smile: Maybe do the same?